src/modules/SourceDialogs.rb

```diff
@@ -1966,11 +1966,10 @@ def ServerWidget
     def IsAnyNetworkAvailable
```

Comment:
We should replace the implementation by just calling [NetworkService.isNetworkRunning](https://github.com/yast/yast-yast2/blob/master/library/network/src/modules/NetworkService.rb#L280) or even drop it and update the callers to use it instead...

Reply:
ok